server/migrations/20220102124147-add-notif-categories.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        return queryInterface.bulkInsert('NotificationCategories', [
+            {
+                name: 'thread-edit',
```

Comment:
Do you have plans to add an emit of these new notification category types in their related edit routes with this PR or another?

Reply:
Adding that in other PR.